modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/friendly/url/test/JournalArticleFriendlyURLTest.java

```diff
@@ -208,10 +208,10 @@ public void testUniqueFriendlyURLForSameTitlesWithExistingArticle()
 		Map<Locale, String> friendlyURLMap = article.getFriendlyURLMap();
 
 		Assert.assertEquals(
-			FriendlyURLNormalizerUtil.normalizeWithEncoding(title + "-2"),
+			FriendlyURLNormalizerUtil.normalizeWithEncoding(title + "-1"),
 			friendlyURLMap.get(LocaleUtil.US));
 		Assert.assertEquals(
-			FriendlyURLNormalizerUtil.normalizeWithEncoding(title + "-2-1"),
+			FriendlyURLNormalizerUtil.normalizeWithEncoding(title + "-1"),
 			friendlyURLMap.get(LocaleUtil.FRANCE));
 	}
```

Comment:
I'm not sure that we want to allow this behavior.

One of the conditions we agreed is that by now allow have the same url in multiple languajes is only aplicable to layouts. Anyway, if  @pabloagulla is ok with this, good to go.

Reply:
Of course, allowing that in all the cases is more consistent and simplifies the logic